mathics/builtin/specialfns/bessel.py

```diff
@@ -667,7 +667,9 @@ class SphericalBesselJ(_Bessel):
 
     attributes = ("Listable", "NumericFunction", "Protected")
 
-    rules = {"SphericalBesselJ[n_, z_]": "Sqrt[Pi / 2] / Sqrt[z] BesselJ[n + 0.5, z]"}
+    rules = {
+        "SphericalBesselJ[n_, z_]": "Times[Rational[1, 2], Power[Pi, Rational[1, 2]], Power[2, Rational[1, 2]], BesselJ[Rational[3, 2], z], Power[Power[z, -1], Rational[1, 2]]]"
```

Comment:
@mmatera I think that adding the new feature shouldn't change this.

Reply:
Yep, this should be in another PR

Reply:
I have removed it form this PR. 

If this kind of rule change is some sort of symbolic manipulation, we should consider implementing this in a more general way so that _whenever_ someone write one expression that rule is manipulated _automatically_ on entry.  One might consider adding a parameter to indicate whether automatic simplification (which I had started to spell symplification) should be applied.